src/test/java/nl/fuchsia/controller/FeitContollerTest.java

```diff
@@ -16,17 +17,24 @@
     @InjectMocks
     FeitContoller feitContoller;
 
-    @Before
+    @BeforeEach
     public void setUp() {
         initMocks(this);
     }
 
-    @Test
+    @Testable
```

Comment:
Waarom gekozen voor @Testable en niet voor @Test?

Reply:
Denk, dat die per ongeluk met omzetten van Junit 4 naar 5 automatisch is aangevuld. Heb hem teruggezet.